app/views/customers/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> Customer Show Page </h1>
+
+<p> <%= @customer.name %> </p>
+
+<h3> Items </h3>
+<ul>
+  <% @customer.items.each do |item| %>
+    <li> <%= item.name %> </li>
+    <li>Price: <%= item.price %> </li>
+    <li>Supermarket: <%= item.supermarket.name %>
+  <% end %>
+</ul>
+
+<h2> Add Item </h2>
+<%= form_with model: [@customer, @customer.customer_items.build], url: customer_add_item_path(@customer), method: :post do |form| %>
```

Comment:
Nice use of form_with model! If you get any weirdness with form behavior in localhost, make sure you disable Turbo